src/runtime/audience-action-flow-test.js

```diff
@@ -148,6 +174,16 @@ describes.realWin('AudienceActionFlow', (env) => {
     sandbox.stub(runtime, 'win').returns(winWithNoGtag);
   }
 
+  async function addToLocalStorage(key, value) {
+    await runtime.storage().set(key, value, true);
```

Comment:
Can we set up an expectation of the return value instead of actually interacting a local storage instance which is suppoed to be mocked?

```
      storageMock
        .expects('get')
        .withExactArgs(...)
        .resolves(...)
        .once();
```